docs/docs-content/troubleshooting/cluster-deployment.md

```diff
@@ -94,6 +95,38 @@ Common reasons for why a service may fail are:
 
 6. Check stdout for errors. You can also open a support ticket. Visit our [support page](http://support.spectrocloud.io/).
 
+## Deployment Violates Pod Security
+Cluster deployment fails with the following message. 
+
+```
+Error creating: pods <name of pod> is forbidden: violates PodSecurity "baseline:v<k8s version>": non-default capabilities …
+```
+
+This can happen when the cluster profile uses Kubernetes 1.25 or later and also includes packs that create Pods requiring elevated privileges . 
+
+### Debug Steps
+
+You can change the Pod Security Standards of the namepace where the Pod is being created to address this issue. 
```

Comment:
```suggestion
To address this issue, you can change the Pod Security Standards of the namespace where the pod is being created. 
```

Reply:
It's up to you how to start the sentence - to me, it just sounds better to start with "To address this issue..."